lib/screens/topic_screen.dart

```diff
@@ -17,14 +17,15 @@ class TopicScreen extends StatefulWidget {
 class _TopicScreenState extends State<TopicScreen> {
   List<Article> _articles;
   bool _isLoading = true;
+  int _currentIndex=0;
 
   void _initTopic() async {
     new ArticleRepo()
         .getArticlesByTopicId(widget.topicId)
         .then((articles) async {
       setState(() {
         _articles = articles;
-        _isLoading = false;
+        _articles.length != 0 ? _isLoading = false: _isLoading = true ;
```

Comment:
if we do not get a response back from the database, but there is no data - the user will see the spinner - confusing him. Better to show nothing in that case

Reply:
Okay sir